server/sonar-server/src/main/java/org/sonar/server/permission/PermissionPrivilegeChecker.java

```diff
@@ -35,16 +36,39 @@ public static void checkGlobalAdminUser(UserSession userSession) {
       .checkPermission(GlobalPermissions.SYSTEM_ADMIN);
   }
 
+  /**
+   * @deprecated does not support organizations. Replaced by {@link #checkProjectAdmin(UserSession, String, Optional)}
+   */
+  @Deprecated
   public static void checkProjectAdminUserByComponentKey(UserSession userSession, @Nullable String componentKey) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)